src/contactCard/Summary.tsx

```diff
@@ -45,7 +54,8 @@ function renderContactSummary(profile: IPersonaProfile, onContactDetailsClick: (
                 <span>{profile.officeLocation}</span>
                 <span>&nbsp;{profile.city}</span>
             </div>
-            <ActionButton className="more-details contact-details" onClick={onContactDetailsClick} aria-label="Show more" aria-expanded={true} tabIndex={0} aria-live="polite">
+            <ActionButton className="more-details contact-details" onClick={() => handleContactDetailsClick(onContactDetailsClick)} aria-label={isExpanded ? 'show more button expanded' : 'show more button collapsed'} aria-expanded={isExpanded} data-focus="button" aria-live="polite">
```

Comment:
aria-expanded already notifies the user if the item is expanded or not, so adding the "show more button expanded" / "show more button collapsed" text to the aria-label will likely result in the screen reader redundantly saying the same information twice